src/main/java/baritone/utils/schematic/format/DefaultSchematicFormats.java

```diff
@@ -67,6 +68,24 @@ public IStaticSchematic parse(InputStream input) throws IOException {
                     throw new UnsupportedOperationException("Unsupported Version of a Sponge Schematic");
             }
         }
+    },
+    /**
+     * The Litematica Schematic Specification. Commonly denoted by the ".litematic" file extension.
+     *
```

Comment:
no spec in the java doc?

Reply:
Litematica has no official specification page I could link to. I opted to reuse the MCEdit comment (I think that's what you mean).